src/main/java/com/webdev/cosmo/cosmobackend/service/external/FacebookClient.java

```diff
@@ -0,0 +1,17 @@
+package com.webdev.cosmo.cosmobackend.service.external;
+
+import com.webdev.cosmo.cosmobackend.security.FacebookUser;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@FeignClient(value = "facebook-client", url = "https://graph.facebook.com/")
+public interface FacebookClient {
+
+    @RequestMapping(method = RequestMethod.GET, value = "/{user-id}")
+    FacebookUser verifyToken(@PathVariable("user-id") String userId,
```

Comment:
rebase with master and stick with one approach, there is already FacebookClient defined